boards/arm/nrf52840_mdk_m2_module/nrf52840_mdk_m2_module.dts

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2022 Matt Anderson
+ * Copyright (c) 2022 Nikola Trifunovic
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ * Copyright (c) 2017 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "nRF52840 MDK M2 Module";
+	compatible = "nrf52840_mdk_m2_module";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0_red: led_0 {
+			gpios = <&gpio0 30 GPIO_ACTIVE_LOW>;
+			label = "Red LED 0";
+		};
+		led0_green: led_1 {
+			gpios = <&gpio0 29 GPIO_ACTIVE_LOW>;
+			label = "Green LED 0";
+		};
+		led0_blue: led_2 {
+			gpios = <&gpio0 31 GPIO_ACTIVE_LOW>;
+			label = "Blue LED 0";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		red_pwm_led: pwm_led_0 {
+			pwms = <&pwm0 30>;
+		};
+		green_pwm_led: pwm_led_1 {
+			pwms = <&pwm0 29>;
+		};
+		blue_pwm_led: pwm_led_2 {
+			pwms = <&pwm0 31>;
+		};
+	};
+
+    /* The button is not present on the module, though reset is typically
+     * connected on the baseboard to reset the module. */
+	buttons {
```

Comment:
please don't mix tabs/spaces

Reply:
fixed in [d0f9bd7](https://github.com/zephyrproject-rtos/zephyr/pull/43992/commits/d0f9bd7f2a6b1c6a706b80ed9617d1cd5b31e48a)